frontend/packages/dev-console/src/components/formik-fields/RadioButtonField.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useField } from 'formik';
+import { FormGroup, Radio } from '@patternfly/react-core';
+import { RadioButtonProps } from './field-types';
+import { getFieldId } from './field-utils';
+
+const RadioButtonField: React.FC<RadioButtonProps> = ({
+  label,
+  value,
+  checked,
+  helpText,
+  required,
+  ...props
+}) => {
+  const [field, { touched, error }] = useField(props.name);
+  const fieldId = getFieldId(value, 'radiobutton');
```

Comment:
getFieldId should be called with props.name not with value.

Reply:
props.name could not be used as props.name for all radio fields is same. using props.name would render same id for all fields, so I used value instead.

Reply:
Got it. 

Now If you are associating multiple controls to one formik field, how you will manage to check/uncheck the radio input controls based on the value?

Reply:
@nemesis09 can you explain why name would be same for all radio fields? We have a similar formik field for checkbox and how it is being used, please take a look at that.

Reply:
@divyanshiGupta  Having different values for the name of the radio fields will enable checking both of them and we don't want that, unlike checkbox. So I used the same name for the radio fields.

Reply:
@gijohn only one radio button from a group can be selected at a time.

Reply:
@divyanshiGupta if you want to read more on Radio buttons, MDN has a pretty good article: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio#Defining_a_radio_group